doc/content/_index.md

```diff
@@ -2,7 +2,12 @@
 title: "Overview"
 description: ""
 weight: 1
-menu:
```

Comment:
Why remove the homepage of the documentation from the main menu?

Reply:
Because it's right next to each other (Home);

<img width="154" alt="Screenshot 2019-07-23 at 16 00 51" src="https://user-images.githubusercontent.com/13334001/61718330-12c86000-ad63-11e9-9e92-6a1bb3e35cd9.png">

The header is not sticky; it scrolls as much out of sight as **Home**

Reply:
To clarify: Home and Overview is the very same thing

Reply:
Ok, but that will change with #250